### ISSUE TITLE
Faker Image

### ISSUE THREAD

**Issue by lagunawebdesign** [fzaninotto/Faker#1281]
Could You remove `curl()` condition in `Provider/Image.php` - just  `$success = copy($url, $filepath);` works great. 
In curl condition there is a problem with `unlink()` function before `fclose()`.

```php
  /*
        // save file
        if (function_exists('curl_exec')) {
            // use cURL
            $fp = fopen($filepath, 'w');
            $ch = curl_init($url);
            curl_setopt($ch, CURLOPT_FILE, $fp);
            $success = curl_exec($ch) && curl_getinfo($ch, CURLINFO_HTTP_CODE) === 200;

            if ($success) {
                fclose($fp);
            } else {
                unlink($filepath);
            }

            curl_close($ch);
        } elseif (ini_get('allow_url_fopen')) {
            // use remote fopen() via copy()
            $success = copy($url, $filepath);
        } else {
            return new \RuntimeException('The image formatter downloads an image from a remote HTTP server. Therefore, it requires that PHP can request remote hosts, either via cURL or fopen()');
        }
        */
        $success = copy($url, $filepath);`

**Comment by fzaninotto** [comment]
> there is a problem with unlink() function before fclose()

Can you be more specific?

**Comment by rino7** [comment]
In my experience, I was getting this error 
`Message: unlink(D:/path/to/facker/img/**e496355.jpg): Permission denied`
I've solved it adding this lines before _CURLOPT_FILE_:
`curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, false); curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
curl_setopt($ch, CURLOPT_FILE, $fp);`

**Comment by fzaninotto** [comment]
Fixed in #1319
